contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,18 +1,36 @@
 * **Objective**:
-    * Expand PostHog’s top of funnel awareness across all our core _individual_ products - product analytics, session recording, feature flags, and A/B testing.
+    * Increase word-of-mouth growth by:
+      1. being an authority and vital resource on building successful products.
+      2. creating a world-class community experience.
```

Comment:
```suggestion
      1. sharing the products and features we've built with the world.
      2. being an authority and vital resource on building successful products.
```
Community hasn't been a focus for us, and this quarter seems to be just planning, so it seems a little weird to have it as a key part of our word of mouth growth.

Something about sharing what we've built ties better to Twitter, internal content, as well as community.

Reply:
Fair. I like this.

Reply:
Maybe we change first one to use the "newsroom of PostHog" given we use it elsewhere?

Reply:
Yes, I do like newsroom